src/autopas/containers/verletClusterLists/VerletClusterLists.h

```diff
@@ -224,12 +225,13 @@ class VerletClusterLists : public ParticleContainerInterface<Particle>, public i
 #ifdef AUTOPAS_OPENMP
 #pragma omp parallel for reduction(|| : deletedSomething)
 #endif
+    // Thanks to clang 13 this has to be a index based loop instead of a range based
     for (size_t i = 0; i < _towerBlock.size(); ++i) {
       auto &tower = _towerBlock[i];
       const auto towerSize = tower.getNumActualParticles();
```

Comment:
```suggestion
      const auto towerSize = tower.size();
```
`getNumActualParticles` would already get number of particles without tailing dummies.